front/app/containers/Admin/users/UserTableRow.tsx

```diff
@@ -42,6 +48,55 @@ interface Props {
   authUser: GetAuthUserChildProps;
 }
 
+const getStatusMessage = (user: IUserData): MessageDescriptor => {
+  const highestRole = user.attributes.highest_role;
+  let statusMessage = messages.registeredUser;
+  if (['admin', 'super_admin'].includes(highestRole)) {
```

Comment:
Maybe an object would look a little bit cleaner here. But no huge difference really, so up to you.

```ts
const roleMessage = {
  admin: messages.platformAdmin,
  super_admin: messages.platformAdmin,
  project_folder_moderator: messages.folderAdmin,
  project_moderator: messages.projectManager,
};

return roleMessage[highestRole];
```

Reply:
Indeed, much cleaner